### PR TITLE
Initial PR for forked repo changes

### PR DESCRIPTION
**Pull request by ekilmer**
This PR should NOT be squashed.

This PR should be rather minimal. Future best-practice PRs will come later, like formatting, linting, type checking, etc.

-------------

Changes references from 'wasm' to 'wasm-tob' or 'wasm_tob' to avoid namespace collisions.

Refactor Python project to use more modern pyproject.toml.

Reset version to 1.0

Add some CI

Adjust README text

Support for all maintained Python versions

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (woodruffw) [inline .github/dependabot.yml:6]: Depending on how often you feel like dealing with these, I suggest lowering this to weekly or every few days 🙂

File: README.md [hunk 2]
```diff
@@ -17,21 +9,36 @@ implemented doesn't correspond to any existing definition and is a
...
+## Installation
+
+```sh
 # From PyPi
-pip install wasm
+pip install wasm-tob
 
 # From GitHub
-pip install git+https://github.com/athre0z/wasm.git
+pip install git+https://github.com/trail-of-forks/wasm-tob.git#egg=wasm-tob
```
Comment (woodruffw) [inline README.md]: The `egg=` fragment is technically non-standard (and I've been working with the `pip` devs to remove it 😅) -- this should be equivalent:

```
pip install git+https://github.com/trail-of-forks/wasm-tob.git
```

(assuming that `wasm-tob` is at the top-level of the repo.)

**Review by woodruffw — APPROVED** [review]
LGTM! Just a few small nits/suggestions.

File: .github/dependabot.yml (hunk 1 above)
Comment (ekilmer) [reply .github/dependabot.yml:6]: I'll leave it daily, and if it gets annoying, I'll change it 👍

File: README.md (hunk 2 above)
Comment (ekilmer) [reply README.md]: Good to know! I'll make the change

**Pull request merged**